authors/apps/articles/views.py

```diff
@@ -591,3 +616,60 @@ def delete(self, request, **kwargs):
                 raise Forbidden
         except Comment.DoesNotExist:
             raise ItemDoesNotExist
+
+
+class CommentOneHistoryView(GenericAPIView):
+    """
+    A view class for handling all operations made to
+    Comment history models directly
+    """
+    permission_classes = [IsAuthenticated]
+    serializer_class = DisplaySingleCommentHistory
```

Comment:
Why do users have to be authenticated to view comment history???

Reply:
History records need to be protected, so that is why I am protecting this get method. I benchmarked from [Github](github.com)

Reply:
There is no need for authentication on this view like Issa said, all non-authenticated users should be able to access the get requests.